Libplanet.Explorer/GraphTypes/ActionType.cs

```diff
@@ -13,13 +17,41 @@ public ActionType()
                 "Arguments",
                 resolve: ctx =>
                 {
+                    object ConvertValue(IValue value)
+                    {
+                        switch (value)
+                        {
+                            case Text text:
+                            case Integer integer:
+                            case Bencodex.Types.Boolean boolean:
+                            case Null n:
+                                return value;
```

Comment:
Shouldn't these be converted to .NET primitive types?

Reply:
Originally, I made the method to avoid treating `Binary` as `IEnumerable`.
I'll add it.